x-pack/plugins/maps/server/routes.js

```diff
@@ -481,6 +484,39 @@ export function initRoutes(router, licenseUid, mapConfig, kbnVersion, logger) {
     }
   );
 
+  router.get(
+    {
+      path: `/${FONTS_API_PATH}/{fontstack}/{range}`,
```

Comment:
If I attempt to cause a 404 by retrieving a font file that doesn't exist (i.e. api/maps/fonts/foo/bar) Kibana throws an internal service error and crashes. @nreese Can you reproduce?